models/account/account.go

```diff
@@ -0,0 +1,106 @@
+package account
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+
+	"github.com/b0d0nne11/scroll-test-project/db"
+	"github.com/mailgun/scroll"
+)
+
+type Account struct {
+	ID   int64
+	Name string
+}
+
+func New(name string) *Account {
+	return &Account{
+		Name: name,
+	}
+}
+
+func (a *Account) Save() error {
+	dbh, _ := db.Get()
+
+	stmt, err := dbh.Prepare("INSERT INTO account (name) VALUES ( ? )")
+	if err != nil {
+		fmt.Printf("error preparing statement: %v\n", err)
+		return err
+	}
+	defer stmt.Close()
+
+	res, err := stmt.Exec(a.Name)
+	if err != nil {
+		fmt.Printf("error creating %v: %v\n", a, err)
+		return err
+	}
+	a.ID, err = res.LastInsertId()
+	if err != nil {
+		fmt.Printf("error creating %v: %v\n", a, err)
+		return err
+	}
+
+	return nil
+}
+
+func GetBy(k string, v string) (*Account, error) {
```

Comment:
Yep that would be very bad. `k` was never passed a user supplied value but the method should never have been public in the first place. I've make it private and only exported `Get(id int64)` and `FindByName(name string)`